pkg/appfile/storage/storage.go

```diff
@@ -1,37 +1,44 @@
 package storage
 
 import (
+	"errors"
+	"os"
+
 	"github.com/oam-dev/kubevela/pkg/appfile/storage/driver"
 )
 
 // Store application store client
 var Store *Storage
 
-func init() {
-	// TODO support provide multiple ways:
-	// system environment
-	// system configfile
-	// startup arguments
-	Store = NewStorage(driver.LocalDriverName)
-}
-
 // Storage is common storage client，use it to get app and others resource
 type Storage struct {
 	driver.Driver
 }
 
+func init() {
+	// default use Local Storage driver
+	_ = NewStorage(driver.LocalDriverName)
+}
+
 // NewStorage form driver type
-func NewStorage(driverName string) *Storage {
-	// TODO remove driverName param ,should use environment get it
-	// FIXME use env to get user storageDriver
+// 1. get system environment from STORAGEDRIVER
+// 2. startup arguments from storage-driver
+func NewStorage(driverName string) error {
+	// first use system environment
+	if storageDriver := os.Getenv("STORAGEDRIVER"); len(storageDriver) != 0 {
```

Comment:
We better use one place to consolidate the ENV and parameters. This will silently override the user input in the flag.

Reply:
OK, I move it to `/cmd/core/main.go`.